src/queries/useAuthoritiesDelete/useAuthorityDelete.js

```diff
@@ -3,15 +3,14 @@ import {
   useQueryClient,
 } from 'react-query';
 
-import {
-  useOkapiKy,
-  useNamespace,
-} from '@folio/stripes/core';
+import { useNamespace } from '@folio/stripes/core';
+
+import { useTenantKy } from '@folio/stripes-authority-components/lib/temp/hooks/useTenantKy';
```

Comment:
Please add this to `stripes-authority-component` exports list, so we can import just from `@folio/stripes-authority-components` path